decorater.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'nameable'
+require_relative 'person'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.capitalize
+  end
+end
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
+  end
+end
+
```

Comment:
| Previous Review 🧾 |
|-|
| ![image](https://github.com/isume295/school-library/assets/93039359/7320b463-2981-4c5d-8f77-ad6b77d6a4ed) |

| Your Readme 📝 |
|-|
| ![image](https://github.com/isume295/school-library/assets/93039359/e1459936-4fae-4892-a8f6-9841f433b19c) |

- [ ] Great work on implementing the changes on code 👍 but I guess you missed this change which the previous reviewer suggested that this project is not Enumerable 😄 So kindly rename it to School Library to make your readme descriptive and professional 😊

Reply:
okay, thank you